misc/vm_configs/scenarios/industry/vm_configurations.h

```diff
@@ -15,8 +15,8 @@
  */
 #define PRE_VM_NUM			0U
 #define SOS_VM_NUM			1U
-#define MAX_POST_VM_NUM			2U
-#define CONFIG_MAX_KATA_VM_NUM		0U
+#define MAX_POST_VM_NUM			7U
```

Comment:
I'm a little confused by these changes (in both the `.h` and `.c` files). Does that mean that the scenario configuration for Industrial was not aligned with what we describe in our documentation?

Reply:
By default and in documentation, industrial scenario supports 1 SOS + 7 post-launched VMs(including 1 KATA VM) which works on most of boards(nuc7i7dnb/WHL/ehl-crb-b boards), so the macro MAX_POST_VM_NUM should be 7, CONFIG_MAX_KATA_VM_NUM should be 1 in the configuration source code(.h, .c) by default. 
Someone has changed the default values for these 2 macros because currently there is a limitation that tgl-rvp only supports 1 SOS + 3 post-launched VMs and he should submit these 2 macros for tgl-rvp which is not generic for most of other boards and aligned with documentation. We currently only submit configuration source code(.h .c) for nuc7i7dnb/whl/ehl and not for tgl-rvp, so I update these macros to align with configuration on nuc7i7dnb/WHL/ehl-crb-b and aligned with our documentation(https://projectacrn.github.io/latest/introduction/index.html#best-known-configurations).

Reply:
Thanks for the explanation, glad we're going back to a default value that's aligned with our documentation. I had missed the patchset that changed this :-(